Integrated feedback from IETF122

wkumari opened the pull request:
I addressed comments from IETF122 IntArea meeting -- see: https://mailarchive.ietf.org/arch/msg/int-area/OV3ZNT4aOBt6oDjLtBYo5TMrdYw/ for summary.

wkumari committed:
Integrated feedback from IETF122

jech commented:
Thanks, Warren.

The bit about recursive changes should go into its own paragraph, since right now the paragraph seems overloaded to me.  Not sure about the bit about the reverse being also possible.

Feel free to either make the fix yourself, or commit as is and let me do the editorial changes.

wkumari commented:
Thank you. I fully agree that the paragraph is clunky as  currently written, but I couldn't figure out how to cleanly break it out. I started down the lines of "The above paragraph notes that the mapping is from prefixes to next-hop pairs; sometimes resolving this next-hop pair first requires resolving an intermediate route - this is referred to as recur... " before deciding that this was clunky...  I'm make the excuse that I'm super-jetlagged at the moment :-) 

I'll commit this. please see if you can massage it into something which flows better. I've requested a slot at the IntArea meeting to hopefully ask for WGLC, and so we should make sure that we post a new version well in advance of the meeting...

jech commented:
Sure.  What's the deadline?

wkumari commented:
I *think* that it is 7/7, but hopefully we can get it in before that...

jech commented:
Ok, I've done some wordsmithing and restructuring.  Mostly editorial.

The one place where I've changed the meaning is that I removed your claim that it's mostly a UI change.  I don't entirely agree, so rather than having an extended discussion on the subject, I suggest we simply drop the claim.

wkumari commented:
Okey dokey, should I post a new version to Datatracker?
W


On Mon, Jul 07, 2025 at 12:50 PM, Juliusz Chroboczek <
***@***.***> wrote:

> *jech* left a comment (wkumari/draft-chroboczek-intarea-v4-via-v6#1)
> <https://github.com/wkumari/draft-chroboczek-intarea-v4-via-v6/pull/1#issuecomment-3045910588>
>
> Ok, I've done some wordsmithing and restructuring. Mostly editorial.
>
> The one place where I've changed the meaning is that I removed your claim
> that it's mostly a UI change. I don't entirely agree, so rather than having
> an extended discussion on the subject, I suggest we simply drop the claim.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/wkumari/draft-chroboczek-intarea-v4-via-v6/pull/1#issuecomment-3045910588>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABDG3XELMM2ICXP2KX3S3AD3HKQONAVCNFSM6AAAAAB76VH3N2VHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZTANBVHEYTANJYHA>
> .
> You are receiving this because you modified the open/close state.Message
> ID: ***@***.***
> com>
>

jech commented:
> Okey dokey, should I post a new version to Datatracker? 

Please do.
